charter.md

```diff
@@ -72,28 +78,156 @@ this works.
 
 ### Vacancies
 
-In the event of a resignation or other loss of an elected steering committee
-member, the candidate with the next most votes from the previous election will
-be offered the seat.  This process will continue until the seat is filled.
+In the event of a resignation or other loss of an elected committee member, the
+next most preferred candidate from the previous election will be offered the
+seat.
+
+A maximum of one (1) committee member may be selected this way between
+elections.
 
 In case this fails to fill the seat, a special election for that position will
-be held as soon as possible. [Eligible voters][voter-eligibility] from the most
-recent election will vote in the special election i.e., eligibility will not be
-redetermined at the time of the special election. A committee member elected in
-a special election will serve out the remainder of the term for the person they
-are replacing, regardless of the length of that remainder.
+be held as soon as possible.
+
+Examples (non-exhaustive):
+
+* the next most preferred candidate is no longer interested in sitting on the
+  committee
+* the next most preferred candidate cannot sit on the committee, due to some
+  committee composition requirement i.e.,
+  [maximal representation][maximal-representation]
 
+[Eligible voters][voter-eligibility] from the most recent election will vote in
+the special election i.e., eligibility will not be redetermined at the time of
+the special election.
+
+A committee member elected in a special election will serve out the remainder
+of the term for the person they are replacing, regardless of the length of that
+remainder.
+
+[maximal-representation]: /elections.md#maximal-representation
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 4 = 1.75
+* 1.75 * 3 = 5.25
+* Round up to the nearest whole number (6)
+* Six (6) affirmative votes would be required to remove a member through a vote
+  of no confidence
+
+The call for a vote of no confidence will happen in a public Steering Committee
+meeting and must be documented as a GitHub issue in the committee's
```

Comment:
> I'd propose stating that the party in question will be permitted to post (in a comment) a link to a defense/rebuttal on their own platform, and that moderation won't delete it.

I can't get behind this either, as it puts restrictions on our moderation and CoC teams.

What I'm worried about is someone using this process/platform to post a manifesto/screed in their defence. I sincerely hope that this wouldn't be the case, but I also hope that this is a policy we don't need to use in the first place.

*If* the impacted person posts a public comment that doesn't violate any moderation or CoC policies, then it can exist just like any other public comment on the issue. I'm not saying they shouldn't be able to do that. My concern is with elevating or somehow enshrining their "right" to do that as some sort of artifact of the process that cannot be touched.

Reply:
An unhinged screed is actually good evidence that removal is correct? I'm not saying we should host it, only that we do people a favor by helping them find it.

Reply:
An unhinged screed likely does undue damage to members of our community. I've seen enough of them in communities in the past to not want a repeat. We don't want to platform or direct people to that.

For any removal situation that wouldn't result in a screed, a comment is welcome and more than enough.

Reply:
Agreed. We have to consider the case of an SC member being removed specifically for CoC violations that include their posts on community-owned channels. In that case, providing them an additional channel to commit a violation on is counterproductive and could cause harm.  And if we include a right to rebuttal in this process document, we won't be able to remove it.

Reply:
> This isn't a trial or a debate.

This looks exactly like a trial with jury comprised of the six other steering members.  Having such a trial without transparent evidence makes it difficult to mount a defense and to have such a trial without the ability to even provide a defense seems even worse.  Claims of "I'm worried about is someone using this process/platform to post a manifesto/screed in their defence" and "providing them an additional channel to commit a violation on is counterproductive and could cause harm" have parallels to previous illegitimate courts such as a those at the Salem witch trials that notably 
1. were created in a rush as a stop-gap ("gave the colony’s legislature the right to establish a court. But the process would take time, and Phips had to act quickly.")
2. made evidence of hearsay ("the admission of rumors, assumptions, and community gossip")
3. prevented challenges to alleged evidence ("most of the witnesses against the accused witches could have been very effectively challenged.")

[source](https://www.history.com/news/salem-witch-trials-justice-legal-legacy#:~:text=The%20second%20key%20legal%20protection,who%20testified%2C%E2%80%9D%20Niehoff%20says.) which concludes these bad examples led to many of guarantees people generally expect from their trials today.  Repeating these errors, for what appears to be similar reasons ("the speedy and vigorous prosecution of such as have rendered themselves obnoxious") is unfortunate.

Reply:
@deads2k please STOP here!

Reply:
> I can assure you that the steering committee talks about everything going on with our community constantly, and the other members will be clear on the position of the person impacted.

The statement provided as justification for the vote of no confidence is not primarily for the other steering members, it's for community members to understand why an attempt to remove their elected representative is happening.

The same is true of the response from the impacted individual. It's for community members, not for other steering members, so the community can hear both sides and give well-informed feedback in the time provided.

> This also doesn't restrict the impacted individual from speaking publicly, only that their statement isn't an official artifact of the process.

The impacted individual's statement should *absolutely* be an official artifact of the process. Refusing to put the accusations and the response side-by-side for community members to weigh, and elevating the accusation while limiting the response to a comment or one of many replies is inappropriate.

I don't find the what-if scenarios about violating moderation or CoC policies a compelling reason to demote the response. The same possibility exists with the accusations, and I'd expect both the accusations and the response to be subject to normal moderation. A response from the impacted person that doesn't violate moderation or CoC policies should be placed side-by-side with the accusations.

Reply:
I wasn't going to say any more on this, but I guess I'll put in one last thought because I see a different primary concern. I don't care about fairness as an end in itself.

The point is for the community to be able to police the board. That is, decide which faction of the board is actually harming the community--both sides will claim the other side is doing so. This policing function is incompatible with only seeing one side of the argument.

There is already social technology for presenting material some people may experience harm from viewing-- content warnings.

To feel why it is important for the community to have this policing role, you can imagine some issue you care greatly about, and imagine it is a tiny minority on the board.

It is very short-sighted to enact rules which can be used against one's "side" if one's "side" falls out of power.

Rules like these are not about being fair to today's incorrect factions. Rules like these are about allowing future correct factions to survive power grabs. I wish it weren't so, but it is.

Reply:
@lavalamp @liggitt @deads2k thanks for your points. I am with @cblecker here. 

```This also doesn't restrict the impacted individual from speaking publicly, only that their statement isn't an official artifact of the process. We don't need to provide a platform.```

I will support the language as-is currently.

Reply:
@justaugustus @parispittman unless you think otherwise, please mark this thread as resolved.

@lavalamp there are plenty of forums for saying one's piece.

Reply:
Given that Steering Committee members are Kubernetes community members, they are covered under what has already been written about the community feedback period:

> There will be a period of two weeks for members of the community to reach
> out to Steering Committee members to provide feedback.
>
> Community members may provide feedback by the following methods:
>
> * commenting on the GitHub issue
> * replying to the notification email
> * sending an email to the
>   [Steering Committee private mailing list][steering-private-list]
> * sending a message to individual committee members

---

> @justaugustus @parispittman unless you think otherwise, please mark this thread as resolved.

Resolving convo.

Reply:
> Steering Committee members are Kubernetes community members, they are covered under what has already been written about the community feedback

Sorry, I really wanted to leave this thread, but I can't with that. I am losing the sense that y'all are arguing in good faith. That is like saying the accused at a trial is in the courtroom and can therefore give feedback like anyone else in the courtroom. The accused doesn't need to give feedback to the board, they are *on* the board already. The accused needs to present a case to the **community**, so that the community can judge the board.

I know you don't think this is like a trial, but I believe you are mistaken about that. I sincerely believe that y'all are having a hard time imagining a hypothetical case where this policy is (in your mind) incorrectly aimed at yourself, and if you took 5 minutes and visualized that as hard as you could, you would not be happy with your options.

I don't feel like my objections have actually been understood and responded to. Policies are foreverish, please consider more than just the current situation when writing them.

Reply:
@lavalamp I do hear your argument, and I assure you that I am approaching this in extremely good faith.

That said, when weighing the potential harms to the community of providing an enshrined platform to someone that the other elected representatives potentially don't have confidence in to serve in the communities best interest, with the potential benefits you've outlined, unfortunately I don't agree with your viewpoint.

Again, to be explicitly clear, we aren't silencing anyone, including the impacted person. If they make comments on the issue or in other public forums that comply with things like our Code of Conduct, those comments can stand like any other. But that impacted person doesn't have a "right" in my view to say whatever they like, irrespective of harm to the community.

I realize that even after all of this, you may still be dissatisfied with this response. However, I do believe that we have fairly and fulsomely discussed this point, even if that resolution isn't the one you hoped for.

Reply:
@justaugustus sure, I'll make a suggestion later today.

Reply:
Thanks Daniel!

Reply:
That was a less trivial task than expected (as always) -- I made a PR directly against Paris's branch: https://github.com/parispittman/steering/pull/1

Reply:
> That was a less trivial task than expected (as always) -- I made a PR directly against Paris's branch: https://github.com/parispittman/steering/pull/1

That PR would alleviate most of my concerns from beginning of this thread.

Reply:
> @lavalamp -- Can you present a solution that you feel is more rounded for all of those involved?
> 
> After 355 comments, in lieu of more discussion, I would love to see a solution in the exact copy you'd like us to consider for inclusion in this charter update.


Not starting a new thread, just looking for clarification on an existing thread.

@justaugustus You requested some specific text and it was provided on the same day.  So far I have not seen your comments here or in https://github.com/parispittman/steering/pull/1.  Do you (and the other steering members) plan to review the text you requested and provide feedback before taking a vote on this PR?  So far, only dims has directly commented.

Reply:
This is a complex topic, and I'd like to thank you both for your engagement and desire to help the community.

I've read and reviewed your proposal, @lavalamp, multiple times over the course of the last week.

When looking at something like this, in the absence of a win-win proposal that satisfies all concerns, those concerns need to be weighted. I believe strongly, and have a track record of advocating for, transparency in open source. Folks should have, whenever possible, the opportunity to know and understand what is happening in the community. What I'm concerned for and weighing this against, is the possibility that an individual who is impacted by this would use this as a loophole to platform harmful views.

We are looking at extreme scenarios here (and I would absolutely classify the use of this process in the first place as an extreme scenario), and I don't deny there is a possibility of a "rogue board" scenario. That said, I think the likelihood of that scenario (six elected committee members, from at least three different employers engaged in a conspiracy to remove the seventh person) is significantly less likely than the "platforming harm" scenario (all it takes is one committee member to execute).

The other piece here that is weighted into my thought process is that the impacted committee member still has the ability to make statements (as a comment on the issue, a link to their gist as you proposed, on their personal blog, twitter, or favourite podcast) without any additional language. The difference as I see it, is that without the proposed revision, those comments are still subject to the Code of Conduct and the CoCC (not SC) can take action if they are harmful, such as moderate and remove them if they are in a project space.

Nobody in our community, even Steering Committee members, and even ones facing forceful removal from the committee, are "above" the code of conduct in my view.

Again, I appreciate your desire to help the community by providing feedback on this, and I assure you I have taken multiple "beats" to consider this opinion and proposed additional language thoughtfully. However, for the reasons above, I'm -1 to modifying the proposed language to include these changes.

Reply:
> The difference as I see it, is that without the proposed revision, those comments are still subject to the Code of Conduct and the CoCC (not SC) can take action if they are harmful, such as moderate and remove them if they are in a project space

I see it the other way around. If we host the artifact it is easy to apply CoC to it? If the person hosts it elsewhere there is little we can do? I didn't intend my proposal to let people post CoC violating subject matter.

Reply:
@lavalamp we take screen shots when we see questionable content go by

Reply:
@dims I don't understand what you mean by that

Reply:
@lavalamp typically when we see what we think is a CoC violation somewhere, we take screen shots just in case it gets cleaned up (by design or by accident). I was responding to the " If the person hosts it elsewhere there is little we can do" in your question.

Reply:
My point is we can't make them take it down / fix it unless we own the forum.

I continue to think you all are overtraining on the possibility of a CoC violating response. I don't see how that would help the person in question at all.

Reply:
@lavalamp those responses rarely help anyone, but they're all too common (just look around at various Big Personalities that have existed in and around linux development communities for decades and the things they've said about queer people/women/disabled people/...).

Reply:
In general, sure, but as a response to an attempt to kick someone off a board? That would make very little sense as self defense. I'll change my mind given a few examples of such a thing, if you have them.

Reply:
@lavalamp no one here is going to take the bait and attempt to change your mind. That isn't positive nor productive. If this community is important enough to you, you are more than enough capable of researching the topic independently.

Reply:
That wasn't disingenuous, but you don't know me and I don't know how to prove that in a short amount of text. Nor do I know how to express the sentiment in a way that sufficiently distinguishes it from people who are attempting to drag out arguments rather than trying to find some objective reason to conclude them, which is how I meant it.

Anyway, it's too late now as people are voting.